README.md

```diff
@@ -2,886 +2,114 @@
 <img src="https://github.com/jina-ai/jina/blob/master/.github/logo-only.gif?raw=true" alt="Jina banner" width="200px">
 </p>
 <p align="center">
-An easier way to build neural search on the cloud
+<h2 align="center">Deep Learning Neural Search Framework for Any Kind of Data</h2>
```

Comment:
Should this sentence case? Personally I think it would read better, but just imo

Reply:
I just checked a few other repo's to see what they do, and some do sentence, some do title. Let's leave it for now